docs/sphinx/compiling/general.rst

```diff
@@ -161,39 +224,32 @@ Cmake build options
     Builds the project with asserts, debug logging and debug symbols.
 * ``-DCMAKE_BUILD_TYPE=Test``
     Builds the project with asserts and debug logging but without debug symbols.
-* ``-DUSE_OLD_DPDK=ON``
-    Builds with DPDK support (<19.11), for source installs requires: `-DRTE_SDK=<dir>`
-    and `-DRTE_TARGET=x86_64-native-linuxapp-gcc`
-    Not needed for newer versions of DPDK on Ubuntu 22.04+.
-    See :ref:`dpdk` for more details.
+* ``-DUSE_DPDK=ON|OFF``
```

Comment:
Should we break up this list of CMAKE options to match that table you sent in slack.  It was kind of nice to see the auto detect vs. optional broken out.